corehq/apps/styleguide/templates/styleguide/bootstrap5/migration_guide.html

```diff
@@ -31,6 +31,7 @@ <h5 class="my-2 ms-3">On this page</h5>
           <li><a href="#un-split-view-files">Un-Split View Files</a></li>
```

Comment:
read the updated guide on staging:
https://staging.commcarehq.org/styleguide/b5/migration/#migrating-report-views